### PR TITLE
 IO page updates for 2.0

### PR DESCRIPTION
**Pull request by kjlyon**
Changes for 2.0 Announcement: 

- Add 2.0 Button
- Shrink `get started` button
- Update scrolly banner text

To view these changes go to: [kjlyon.github.io/snap](https://kjlyon.github.io/snap/)

@intelsdi-x/snap-maintainers

### CODE REVIEW

File: zurb-template/src/data/event.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [ 
 { 
-"text": "Snap and Kubernetes Integration for better monitoring and scale", 
-"link": "https://medium.com/intel-sdi/how-we-integrated-kubernetes-with-snap-and-come-up-with-kubesnap-466ac25fc0ff#.61rotoqqw" 
+"text": "Snap plugin diagnostics allows for easier debugging", 
+"link": "https://github.com/intelsdi-x/snap/blob/docs/PLUGIN_DIAGNOSTICS.md"
```
Comment (katarzyna-z) [inline zurb-template/src/data/event.json]: The link to PLUGIN_DIAGNOSTICS.md does not work. It seems that you should change the address to https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_DIAGNOSTICS.md

File: zurb-template/src/data/event.json [hunk 2]
```diff
@@ -15,18 +15,18 @@
 },
 
 { 
-"text": "Set single or multiple runs for task with count parameter", 
-"link": "https://github.com/intelsdi-x/snap/blob/master/docs/TASKS.md#task-manifest" 
+"text": "2.0 release introduces a new plugin type: streaming collectors", 
+"link": "https://github.com/intelsdi-x/snap/blob/master/docs/STREAMING.md"
```
Comment (katarzyna-z) [inline zurb-template/src/data/event.json]: For now this link does not work. Documentation for streaming is in code review.

File: zurb-template/src/data/event.json [hunk 3]
```diff
@@ -15,18 +15,18 @@
...
-"link": "https://github.com/intelsdi-x/snap/blob/master/docs/TASKS.md#task-manifest" 
+"text": "2.0 release introduces a new plugin type: streaming collectors", 
+"link": "https://github.com/intelsdi-x/snap/blob/master/docs/STREAMING.md" 
 },
 
 { 
-"text": "Plugin Authors: Go RPC now deprecated", 
-"link": "https://github.com/intelsdi-x/snap/issues/1289"
+"text": "Plugin Authors: \"Supported\" plugins tag indicates premier plugins", 
+"link": "https://github.com/intelsdi-x/snap/blob/docs/PLUGIN_STATUS.md"
```
Comment (katarzyna-z) [inline zurb-template/src/data/event.json]: This link also does not work. You should change the address to https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_STATUS.md

File: zurb-template/src/pages/index.html [hunk 4]
```diff
@@ -17,11 +17,18 @@ <h3 class = "subheader center"> Easily collect, process, and publish telemetry d
 
 
       <div class = "row">
-      <div class="large-12 columns center">
-        <a class="button" href="https://github.com/intelsdi-x/snap#getting-started">get started</a>
+      <div class="large-12 columns  center">
+          <a class="button button2" href="https://github.com/intelsdi-x/snap/releases">snap reaches 2.0!</a>
```
Comment (katarzyna-z) [inline zurb-template/src/pages/index.html]: I think that we should write "Snap" instead of "snap".

File: zurb-template/src/data/event.json (hunk 2 above)
Comment (kjlyon) [reply zurb-template/src/data/event.json]: This link works now that the documentation has been merged.

**Comment by kjlyon** [comment]
Thank you for review @katarzyna-z , I updated my PR. I will wait until Monday to merge this.

**Pull request merged**